core/templates/rid_owner.h

```diff
@@ -163,7 +163,7 @@ class RID_Alloc : public RID_AllocBase {
 		return _allocate_rid();
 	}
 
-	_FORCE_INLINE_ T *get_or_null(const RID &p_rid, bool p_initialize = false) {
+	_FORCE_INLINE_ T *get_or_null(const RID &p_rid, bool p_initialize = false) const {
```

Comment:
Not sure about this one, it's not a constant function as it does mutate things, so using `const` here is misleading, I'm not sure why it even compiles safely tbh

Many of the owners are marked as `mutable` so I'd leave this alone and make the ones that need this mutable instead

Reply:
the name led me to believe it was like a const function, and it does behave that way if `p_initialize` is false, the default arg, but i understand ill just put mutable in one place

(it compiles bc `chunks` points to mutable memory)